modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaLayoutServicesCheck.java

```diff
@@ -0,0 +1,155 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.source.formatter.check;
+
+import com.liferay.petra.string.StringBundler;
+import com.liferay.portal.kernel.util.ArrayUtil;
+import com.liferay.portal.kernel.util.StringUtil;
+import com.liferay.source.formatter.check.util.JavaSourceUtil;
+import com.liferay.source.formatter.parser.JavaClass;
+import com.liferay.source.formatter.parser.JavaClassParser;
+import com.liferay.source.formatter.parser.JavaMethod;
+import com.liferay.source.formatter.parser.JavaTerm;
+
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * @author Tamyris Bernardo
+ */
+public class UpgradeJavaLayoutServicesCheck extends BaseUpgradeCheck {
+
+	public boolean validateParameters(
+		String fileName, String javaMethodContent, List<String> parameterList,
+		String[] parameterTypes) {
+
+		if (!hasParameterTypes(
+				javaMethodContent, javaMethodContent,
+				ArrayUtil.toStringArray(parameterList), parameterTypes)) {
+
+			addMessage(
+				fileName,
+				StringBundler.concat(
+					"Could not conclude upgrade of one the addLayout() or ",
+					"updateLayout() methods. The method signature has changed ",
+					"with new parameters. Fill the new parameters manually."));
```

Comment:
@nicolas-sss, @TamyrisTorres, let's rephrase to:
"Unable to format methods addLayout and updateLayout from LayoutService and LayoutLocalService, fill the new parameters manually."

Please see https://liferay.atlassian.net/wiki/spaces/ENGR/pages/1511752578/Wordsmith+conventions+in+Liferay

In addition, can't we add a link to the documentation generated by SF in this pull? It would be great from the perspective of the developer reading this message and it'll also help him/her to fix the problem. :wink: Let's produce excellence, folks!